src/vivarium/framework/values/pipeline.py

```diff
@@ -266,21 +246,11 @@ def set_attributes(
         manager
             The simulation values manager.
         """
-        self._set_common_attributes(component, combiner, post_processor, manager)
-        self.source = ValueSource(self, source, component)
-
-    def _set_common_attributes(
-        self,
-        component: Component | None,
-        combiner: ValueCombiner,
-        post_processor: PostProcessor | AttributePostProcessor | None,
-        manager: ValuesManager,
-    ) -> None:
-        """Set the common attributes for all pipeline types."""
         self.component = component
         self._combiner = combiner
         self.post_processor = post_processor
         self._manager = manager
+        self.source = ValueSource(self, source, component)
```

Comment:
Nit: wasn't this the first attribute set before?

Reply:
Second, actually. I moved it back.